Patch by Robert Scheck <robert@fedoraproject.org> for Zarafa <= 7.1.8 which ensures that
the removed man pages for proprietary Zarafa Web Services are not tried to be installed.

--- zarafa-7.1.8/doc/Makefile.am			2014-02-20 12:02:44.000000000 +0100
+++ zarafa-7.1.8/doc/Makefile.am.man-pages		2014-02-21 08:55:04.000000000 +0100
@@ -8,7 +8,11 @@
 	zarafa-ical.cfg.5 zarafa-server.cfg.5 zarafa-ldap.cfg.5 zarafa-ldapms.cfg.5 zarafa-unix.cfg.5 zarafa-spooler.cfg.5 zarafa-gateway.cfg.5 \
 	zarafa-monitor.cfg.5 zarafa-licensed.cfg.5 zarafa-dagent.cfg.5 zarafa-search.cfg.5 zarafa-archiver.1 zarafa-archiver.cfg.5 \
 	zarafa-msr.1 zarafa-msr.cfg.5 zarafa-msr-verify.1 za-aclsync.1 za-aclset.1 zarafa-backup.cfg.5 za-restore.1 \
-	zarafa-set-oof.1 zarafa-mailbox-permissions.1 zarafa-autorespond.1 zarafa-ws.1 zarafa-ws.cfg.5
+	zarafa-set-oof.1 zarafa-mailbox-permissions.1 zarafa-autorespond.1
+
+if ! OSS_ONLY
+allpages += zarafa-ws.1 zarafa-ws.cfg.5
+endif
 
 CLEANFILES = zarafa-cfgchecker.1
 
--- zarafa-7.1.8/doc/Makefile.in			2014-02-20 12:08:17.000000000 +0100
+++ zarafa-7.1.8/doc/Makefile.in.man-pages		2014-02-21 08:57:00.000000000 +0100
@@ -294,7 +294,8 @@
 	zarafa-ical.cfg.5 zarafa-server.cfg.5 zarafa-ldap.cfg.5 zarafa-ldapms.cfg.5 zarafa-unix.cfg.5 zarafa-spooler.cfg.5 zarafa-gateway.cfg.5 \
 	zarafa-monitor.cfg.5 zarafa-licensed.cfg.5 zarafa-dagent.cfg.5 zarafa-search.cfg.5 zarafa-archiver.1 zarafa-archiver.cfg.5 \
 	zarafa-msr.1 zarafa-msr.cfg.5 zarafa-msr-verify.1 za-aclsync.1 za-aclset.1 zarafa-backup.cfg.5 za-restore.1 \
-	zarafa-set-oof.1 zarafa-mailbox-permissions.1 zarafa-autorespond.1 zarafa-ws.1 zarafa-ws.cfg.5
+	zarafa-set-oof.1 zarafa-mailbox-permissions.1 zarafa-autorespond.1
+@OSS_ONLY_FALSE@allpages += zarafa-ws.1 zarafa-ws.cfg.5
 
 CLEANFILES = zarafa-cfgchecker.1
 man_MANS = $(BUILT_SOURCES) $(allpages)
